discovery-infra/start_discovery.py

```diff
@@ -330,6 +330,13 @@ def _get_provisioning_cidr(cidr, ns_index):
     provisioning_cidr += ns_index + consts.NAMESPACE_POOL_SIZE
     return str(provisioning_cidr)
 
+def _get_provisioning_cidr6(cidr):
```

Comment:
Maybe support multi deployments with ns_index?

Reply:
The cidr should be already based on ns_index.

Reply:
iirc, @RazRegev added this so the CI can run simultaneously, every run runs with the same cidr.

Reply:
OK, will add